pkg/loop/server.go

```diff
@@ -90,6 +106,27 @@ func (s *Server) start() error {
 		return fmt.Errorf("error starting health checker: %w", err)
 	}
 
+	if envCfg.DatabaseURL != nil {
```

Comment:
is the same db username being used in core?

Reply:
This code doesn't have an opinion on that and should be flexible enough to work with any changes on the core side.

Reply:
Yes, but what do you mean by locked out? Currently, only the core host holds a lease lock, but that could change in the future.

Reply:
I've also been wondering if we need to do anything special to handle the lease locks for LOOP plugins. As I understand it, the lease lock is only advisory... so for now the LOOP can just connect to the db and ignore it.  However, it seems like that might leave the door open for two different LOOP plugins of the same type clobbering each other if somehow they both get pointed at the same db.

Maybe this can't happen, because they would have had to have been passed the db url from two different chainlink nodes configured for the same db... and whichever one doesn't get the lock would abort before ever starting a LOOP plugin?

Alternatively, is there any way the same chainlink node could end up starting two LOOP plugins of the same type? What if it starts one, then gets restarted itself but for some reason the first LOOP plugin executable is still running, and then it starts another one? I'm guessing that would cause more problems than just the db.

Reply:
> What if it starts one, then gets restarted itself but for some reason the first LOOP plugin executable is still running

Seems like this would have to be possible if the chainlink executable received a SIGKILL instead of a SIGTERM, but then probably the second LOOP plugin wouldn't start due to port conflicts?

Reply:
I agree we will have to consider these things and should do defensive locking, but I'm not sure we can address it yet in the short term until we establish stricter isolation :thinking: